npm/private/npm_translate_lock.bzl

```diff
@@ -80,6 +80,7 @@ _ATTRS = {
     "prod": attr.bool(),
     "public_hoist_packages": attr.string_list_dict(),
     "quiet": attr.bool(default = True),
+    # TODO(bzlmod): remove repositories_bzl_filename
```

Comment:
Do you mean remove when we drop support for WORKSPACE?

Reply:
Yeah, do we have a keyword I should put here instead of `bzlmod`?